src/EVault/modules/Governance.sol

```diff
@@ -224,7 +229,8 @@ abstract contract GovernanceModule is IGovernance, Base, BalanceUtils, BorrowUti
             newLTV.targetTimestamp,
             newLTV.targetLTV.toUint16(),
             newLTV.rampDuration,
-            newLTV.originalLTV.toUint16()
+            newLTV.originalLTV.toUint16(),
+            !origLTV.initialized
```

Comment:
this will return false after first initialization, maybe rename the parameter in the event into something like: `wasInitialized` or something like that.